js/index.js

```diff
@@ -1 +1,77 @@
 // Your code goes here
```

Comment:
I'm not sure if you needed to use 10 different events but at this point, I'm not concerned because you've demonstrated you know how to act upon the core concept that was taught in this lesson. 

I wished this one was as clean and organized as your DOM I project. I recommend in the future to  add a comment above each thing and just leaving a little note about what you're expecting it to do. This is a good habit to help future developers that are told to work on your code. It makes debugging and refactoring easy too. Overall you did a great though on the project.